homeassistant/components/esphome/entry_data.py

```diff
@@ -48,6 +48,7 @@
 from aioesphomeapi.model import ButtonInfo
 from bleak_esphome.backend.device import ESPHomeBluetoothDevice
 
+from homeassistant.components.assist_satellite import AssistSatelliteConfiguration
```

Comment:
The Assist Satellite models are not meant to be used outside the Assist Satellite platforms. If needed, the esphome integration should use an internal model for data that is passed to other parts of the integration from the platform.